bootstrap/main.go

```diff
@@ -111,6 +111,21 @@ func mainImpl() {
 		}
 	}
 
+	// TODO: Should we check always?
```

Comment:
IMO we should always check to make sure DNS is working as accepted, even with GKE. WDYT?

Reply:
I agree!